ehrql/query_model/table_schema.py

```diff
@@ -41,6 +41,15 @@ def __post_init__(self):
         def description(self):
             return f"Matches regular expression: `{self.regex}`"
 
+    class Range(BaseConstraint):
+        minimum: int
+        maximum: int
+        step: int = 1
+
+        @property
+        def description(self):
+            return f"Must be >= {self.minimum}, <= {self.maximum}, and a multiple of {self.step}"
```

Comment:
I think the description moods have changed since I first wrote that ticket (see the other constraint descriptions) so for consistency this should use something like "Always" rather than "Must be".

I also wonder if we should get rid of the "multiple of" bit if `step` is 1 as that's going to look a bit weird.

If you do end up adding logic to the `description` property then you might find coverage moaning at you, in which case there are already some unit tests for the other constraints which you could add to:
https://github.com/opensafely-core/ehrql/blob/94fea127a67d824c438bcaecf7927834b40d8c4e/tests/unit/query_model/test_table_schema.py#L105-L116